.buildkite/auditbeat/scripts/package.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/env-scripts/linux-env.sh
+
+echo ":: Docker Version ::"
```

Comment:
Would it be easier for the BK output to have something like `echo "--- Docker Version: $(docker --version)"`?